pkgs/build-support/kernel/make-initrd-ng/src/main.rs

```diff
@@ -0,0 +1,207 @@
+use std::collections::{HashSet, VecDeque};
```

Comment:
A high-level description of what this tool is doing would be good to add here. It doesn't do a lot and you can grasp it by reading through it but the signal/noise ratio isn't that great. Helps future readers/debuggers to have a "one paragraph" idea of what goes in and out of here.

Reply:
Agreed